WORKSPACE

```diff
@@ -4,6 +4,8 @@ load("//third_party:deps.bzl", "deps")
 
 deps()
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
ok, that one is not needed. An oversight on my part. It is imported below already.